lib/db/main.js

```diff
@@ -106,6 +106,17 @@ const optimize = (conn, database, params) => {
   }).then(httpBody);
 };
 
+const copy = (conn, database, destination, params) => {
```

Comment:
What's this for?

Reply:
Are you looking at a specific commit? This change shouldn't be in the PR (and I don't see it there). I had accidentally removed the `copy` method and then added it back.

Reply:
Oops, just realized the last commit after